content/en/blog/_posts/2023-04-11-kmsv2-beta.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: blog
+title: "Kubernetes 1.27: KMS V2 Moves to Beta"
+date: 2023-04-11
+slug: kms-v2-moves-to-beta
+---
+
+**Authors:** TBD
+
+With Kubernetes 1.27, we are moving Key Management Service (KMS) v2 API to beta.
+
+## What is KMS?
+One of the first things to consider when securing a Kubernetes cluster is encrypting etcd data at rest. KMS provides an interface for a provider to utilize a key stored in an external key service to perform this encryption.
```

Comment:
Please **manually wrap** the long lines like this for ease of review and for ease of change tracking, especially for downstream localization teams. I.e.,

```suggestion

One of the first things to consider when securing a Kubernetes cluster is encrypting etcd data at rest.
KMS provides an interface for a provider to utilize a key stored in an external key service
to perform this encryption.
```